### PR TITLE
send debug info to output window if we are remote debugging.

### LINKED ISSUE #3057: Capturing output of the remote application
When an application is launched remotely using the VS C# project system, the output/error is not currently available. 

Setting 'VsDebugTargetInfo4.fSendToOutputWindow = TRUE;` will capture the output from the remotely launched application and display it int he VS Output window. 

related issue: https://github.com/dotnet/project-system/issues/3053

### PR DESCRIPTION
**Pull request by jmarolf**
fixes #3057

### CODE REVIEW

File: src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Debug/ProjectDebuggerProvider.cs [hunk 1]
```diff
@@ -269,9 +269,18 @@ internal static VsDebugTargetInfo4 GetDebuggerStruct4(IDebugLaunchSettings info)
             debugInfo.AppPackageLaunchInfo = info.AppPackageLaunchInfo;
             debugInfo.project = info.Project;
 
+            if (IsRemoteDebugging(info))
```
Comment (davkean) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Debug/ProjectDebuggerProvider.cs]: The legacy project system does this based on when the app is "WinExe", why is this different? 

See CVsProjDebugLaunch.cpp:940, this value eventually flows down to the debug info.

Comment (davkean) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Debug/ProjectDebuggerProvider.cs]: Or perhaps this bug was supposed to be against the legacy project system? @rajkumar42 was #3057 intended for legacy or new project system, or both?

Comment (davkean) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Debug/ProjectDebuggerProvider.cs]: Ping?

Comment (jmarolf) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Debug/ProjectDebuggerProvider.cs]: I need to setup a remote test environment to confirm.  I agree that this is not the right fix.

Comment (rajkumar42) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Debug/ProjectDebuggerProvider.cs]: @davkean apologies, I don't know the difference between legacy vs. the new project system. 'fSendToOutputWindow' was a relatively new feature - so I don't think it is in any project system yet (except for Docker).

**Pull request closed without merging**